src/Mod/Draft/draftguitools/gui_trimex.py

```diff
@@ -153,7 +151,7 @@ def proceed(self):
                 if isinstance(e.Curve,(Part.BSplineCurve, Part.BezierCurve)):
                     self.obj = None
                     self.finish()
-                    _err(translate("draft", "Trimex is not supported yet on this type of object."))
+                    _err(translate("draft", "Trimex is not supported yet on this type of object"))
```

Comment:
"Trimex is not supported yet on this type of object"
->
"Trimex does not support this object type"